packages/massa-web3/src/experimental/smartContract.ts

```diff
@@ -278,6 +288,7 @@ export class SmartContract {
       throw new Error(result.info.error)
     }
 
-    return BigInt(result.info.gasCost)
+    // eslint-disable-next-line @typescript-eslint/no-magic-numbers
+    return (BigInt(result.info.gasCost) * 120n) / 100n
```

Comment:
is result.info.gasCost already a number?
if yes:
```suggestion
    return BigInt(result.info.gasCost * 1.2)
```